resources/js/components/CrossSells.vue

```diff
@@ -0,0 +1,23 @@
+<script>
+export default {
+    render() {
+        return this.$scopedSlots.default(this)
+    },
+
+    data: () => ({
+        amastyIds: []
+    }),
+
+    async mounted() {
+        this.amastyIds = await rapidezAPI('post', 'cart/cross-sells', {
+            ids: this.cartIds
+        })
+    },
+
+    computed: {
+        cartIds() {
```

Comment:
I think product ids would make more sense here, as it's not the quote ids we're returning here